sources/packages/web/src/components/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <v-form ref="approveApplicationOfferingChangeRequest">
+    <modal-dialog-base
+      :showDialog="showDialog"
+      title="StudentAid BC Declaration"
+    >
+      <template #content>
+        <error-summary
+          :errors="approveApplicationOfferingChangeRequest.errors"
+        />
+        <p>
+          Please review and accept the StudentAid BC declaration to allow the
+          institution to make proposed changes to your application (on your
+          behalf).
+        </p>
+        <content-group-info class="div-scroll"
```

Comment:
is the plan to keep the small, since the content is changed?

Reply:
Keeping it the same size as discussed yesterday.